src/basics.md

```diff
@@ -470,6 +471,80 @@ fn main() {
 }
 ```
 
+[ex-phone]: #ex-phone
+<a name="ex-phone"></a>
+## Extract phone numbers from text
+
+Attempts to process a static string as a list including US phone numbers.  The
```

Comment:
This sentence looks a little off, I wouldn't say that the code "attempts" it actually does the parsing. Also I would keep the whole description as a short one sentence mentioning the key method used (`captures_iter`).